CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+docs/CONTRIBUTING.md
```

Comment:
ok super confusing but apparently this is how GH renders symlinks in diffs?
See screenshot from viewing [the commit](https://github.com/argoproj/argo-cd/blob/c33ce38c3a980bf6bcfb7325abada71d7e13ffe1/CONTRIBUTING.md) which says "Symbolic link":
![Screenshot 2023-08-03 at 5 13 02 PM](https://github.com/argoproj/argo-cd/assets/4970083/c5a3d82d-965c-4b04-8d14-a63923a56fae)

Reply:
GitHub handles the symlink poorly. Looking at the tree on the commit, in the Preview view, it just displays the path. On the code view, I can click "Symbolic Link" to go to the file under `docs/`. 

I think it makes more sense to copy and paste what's in `docs/CONTRIBUTING.md` instead. At least that way, it's less steps for users to get to [the Contribution Guide](https://argo-cd.readthedocs.io/en/latest/developer-guide/code-contributions/).

```suggestion
Please refer to [the Contribution Guide](https://argo-cd.readthedocs.io/en/latest/developer-guide/code-contributions/)
```

Reply:
Yea, I was hoping GitHub would redirect it to the other file or render the other file (similar to how symlinks behave in usage), but it apparently does neither 😕

Was hoping to avoid copy+paste since that comes with synchronization maintenance, but guess that can't be avoided here. This file does not change frequently at least

Reply:
Changed to pure `cp`